chapters/syntax.tex

```diff
@@ -57,18 +57,17 @@ \section{Lexical conventions}\label{lexical-conventions}
   Modelica uses the same comment syntax as C++ and Java (i.e., \lstinline!//! signals the start of a line comment and \lstinline!/* $\ldots$ */! is a multi-line comment); comments may contain any Unicode character.
   Modelica also has structured comments in the form of annotations and string comments.
 \item
-  Each description-string or string in annotations (= \lstinline[language=grammar]!STRING! with production annotation-clause in the grammar) may contain any member of the Unicode character set.
+  Each \lstinline[language=grammar]!description-string! or string in annotations (that is, a \lstinline[language=grammar]!STRING! with production \lstinline[language=grammar]!annotation-clause! in the grammar) may contain any member of the Unicode character set.
   All other strings have to contain only the subset of Unicode characters identical with the 7-bit US-ASCII character set.
   \begin{nonnormative}
   As a consequence, operators like `\lstinline!>!' or `\lstinline!<!', and external functions only operate on ASCII strings and not on Unicode-strings.
-  Within a description-string the tags \lstinline!<HTML>! and \lstinline!</HTML>! or \lstinline!<html>! and \lstinline!</html>! define optionally begin and end of content that is HTML encoded.
+  Within a \lstinline[language=grammar]!description-string! the tags \lstinline!<HTML>! and \lstinline!</HTML>! or \lstinline!<html>! and \lstinline!</html>! define optionally begin and end of content that is HTML encoded.
```

Comment:
Looks like a consequence of duplicated statements in the specification, compare #2672.

In [18.2.1 _Class Description and Revision History_](https://specification.modelica.org/master/annotations.html#annotation-info-revisions) we have this:
> If the string starts with the tag &lt;html> or &lt;HTML> the entire string is HTML encoded (and is assumed to end with &lt;/html> or &lt;/HTML> and shall be rendered as HTML even if the end-tags are missing), …

Isn't that enough?  Can't we simply drop the entire sentence from the _syntax.tex_?

Reply:
The sentence in _syntax.tex_ covers both description strings and annotations. The one in 18.2.1 covers only the annotations. Maybe the better option is to move the description from 18.2.1 to _syntax.tex_ (assuming it holds for description strings as well). And in 18.2.1 just put "For HTML tags, see" the same way there is already a reference for external links there.

Reply:
So, either do nothing or drop from annotations. And apply suggested change above.

Reply:
While we have given lots of attention to HTML in `Documentation`, the description strings should be considered being in the same undeveloped state that HTML in `Documentation` was at the time of reporting #2593.

I also find it problematic that _syntax.tex_ contains an old statement about annotations in general, whereas our refined formulations in _annotations.tex_ only applies to `Documentation`.  Are there even any other annotations where HTML tags are supported?  In exactly which forms should they be recognized in the other annotations?

Considering that a change/clarification of how HTML tags may be used in description strings should be approved by the language group, and the need to clarify the relation between `Documentation` and annotations in general, I think there are enough problems with this item to motivate opening a separate issue rather than touching it in this PR.

Reply:
Agreed that it deserves its own issue.